import "MItem.proto";

message DseSpiritDebrisList
{
	repeated MItem				items = 1;	//碎片列表
}
